java/src/main/java/org/upgradeplatform/responsebeans/InitRequest.java

```diff
@@ -12,10 +12,18 @@
 	private String id;
 	private Map<String, List<String>> group;
 	private Map<String, String> workingGroup;
-	
+	private InitRequest originalUser;
```

Comment:
This recursive inclusion of `InitRequest` inside itself doesn't make sense to me.  What is it for?

Reply:
@VivekFitkariwala 
When we set alternative userIds, the response object contains  a field `originalUser`.  I have added it to initRequest class so that we don't have to create new class.

Example response object: 
`{
        "id": "user1alias1",
        "originalUser": {
            "createdAt": "2020-06-11T07:10:21.583Z",
            "updatedAt": "2020-06-11T07:10:21.583Z",
            "versionNumber": 1,
            "id": "user1",
            "group": {
                "teachers": [
                    "1",
                    "2",
                    "7"
                ],
                "classes": [
                    "1",
                    "2",
                    "3",
                    "4",
                    "5"
                ]
            },
            "workingGroup": null
        },
        "group": null,
        "workingGroup": null,
        "createdAt": "2020-06-12T02:55:42.261Z",
        "updatedAt": "2020-06-12T02:55:42.261Z",
        "versionNumber": 1
    }`

Reply:
Whole document of original user might be redundant here. We can just send this data in response.
```
{
        "id": "user1alias1",
        "originalUser": "user1",
        "group": null,
        "workingGroup": null,
        "createdAt": "2020-06-12T02:55:42.261Z",
        "updatedAt": "2020-06-12T02:55:42.261Z",
        "versionNumber": 1
    }
```